src/content/docs/infrastructure/host-integrations/host-integrations-list/mongodb/mongodb-migration-guide.mdx

```diff
@@ -0,0 +1,316 @@
+---
+title: MongoDB monitoring migration guide
+tags:
+  - Integrations
+  - On-host integrations
+  - On-host integrations list
+metaDescription: "New Relic's MongoDB integration: how migrate from legacy to to the new version."
+---
+
+The existing legacy version will be deprecated and no longer supported during FY24.
+
+### Why do I need to migrate to the new version?
+
+The new version brings these benefits in comparison with the legacy version:
+
+* Support of the latest major MongoDB versions 5 and 6.
+* Compatibility with [Percona Server for MongoDB](https://www.percona.com/software/mongodb/percona-server-for-mongodb).
+* Compatibility with [MongoDB Atlas](https://www.mongodb.com/cloud/atlas/efficiency).
```

Comment:
here we say that is supported, in the other page it is marked as "Coming soon"

Reply:
Thanks, already fixed in my previous commit.